src/app/all-collections/all-collections.component.ts

```diff
@@ -28,9 +28,7 @@ export class AllCollectionsComponent implements OnInit {
   seriesPagination!: Pagination;
   collectionTagActions: ActionItem<CollectionTag>[] = [];
 
-  constructor(private collectionService: CollectionTagService, private router: Router, 
-    private route: ActivatedRoute, private seriesService: SeriesService, private toastr: ToastrService, 
-    private actionFactoryService: ActionFactoryService, private modalService: NgbModal) {
+  constructor(private collectionService: CollectionTagService, private router: Router, private route: ActivatedRoute, private seriesService: SeriesService, private toastr: ToastrService, private actionFactoryService: ActionFactoryService, private modalService: NgbModal) {
```

Comment:
Just for next time, usually you want the constructor to be split up so it's easier to see instead of one long line. You don't have to change it in this PR.